fenix/app/src/main/res/values/strings.xml

```diff
@@ -610,6 +610,8 @@
 
     <!-- Preference for add_ons -->
     <string name="preferences_addons">Add-ons</string>
+    <!-- Preference for installing a local add-on -->
+    <string name="preferences_install_local_addon">Install local Addon</string>
```

Comment:
```suggestion
    <string name="preferences_install_local_addon">Install add-on from file</string>
```

to match what we have in `about:addons`:

<img width="387" alt="Screenshot 2023-11-23 at 09 39 21" src="https://github.com/mozilla-mobile/firefox-android/assets/217628/0cb12f32-aa70-4e32-aede-129aa835d382">